docs/balena-cli.md

```diff
@@ -333,6 +333,30 @@ Examples:
 
 ### Options
 
+#### --fields FIELDS
+
+only show provided fields (comma-separated)
+
+#### -j, --json
+
+output in json format
+
+#### --filter FILTER
+
+filter results by substring matching of a given field, eg: --filter field=foo
+
+#### --no-header
```

Comment:
Let's change the title to `fleets: Add --filter, --no-header, --no-truncate, and --sort options` so that we are consistent with the older changelog entries.